src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -297,6 +297,12 @@ private static string GetNextMethodName(ArrayBuilder<MethodSymbol> builder)
                             var typeNameDecoder = new EETypeNameDecoder(Compilation, (PEModuleSymbol)_currentFrame.ContainingModule);
                             foreach (var alias in aliases)
                             {
+                                if (alias.IsReturnValue0())
+                                {
+                                    Debug.Assert(aliases.Where(a => a.Kind == DkmClrAliasKind.ReturnValue).Count() > 1);
```

Comment:
I like the predicate to Count one, but I don't think Any has the same semantics (that would be ```>= 1``` vs ```> 1```)